src/screens/profile-screen/ProfileTabNavigator.tsx

```diff
@@ -27,37 +36,67 @@ const CollectiblesTab = () => {
   return <Text>Collectibles Tab</Text>
 }
 
-export const ProfileTabNavigator = () => {
+type ProfileTabNavigatiorProps = {
+  profile: ProfileUser
+}
+
+export const ProfileTabNavigator = ({ profile }: ProfileTabNavigatiorProps) => {
+  const { neutral } = useThemeColors()
+  const isArtist = profile.track_count > 0
+
+  const showCollectiblesTab = useShouldShowCollectiblesTab(profile)
+
+  const IconRepostFixed = useCallback(() => <IconRepost fill={neutral} />, [
```

Comment:
Why is this only needed for the repost icon?

Reply:
for some reason the icon repost svg wasn't picking up the fill, was seeing: 
![image](https://user-images.githubusercontent.com/8230000/152609744-5085fe38-9879-425d-bdff-d30169907e0f.png)
any thoughts there?

Reply:
It looks like it's because we have the fill defined as `#000` in the repost icon, so we can use the fill prop in other places via react-native-svg. But react-navigation is styling the svgs differently, so the ones that have an actual color defined and aren't being overridden work. 

We should agree on one method of coloring svgs, I'm for react-native-svg (mainly because I don't know how the other method works. Can we just define a fill in the styles?)

Reply:
updated all the icons, and using the provided "color" prop for the icon option, thanks for your help on this